bitnami/wavefront/templates/collector-config.yaml

```diff
@@ -0,0 +1,190 @@
+{{- if .Values.collector.enabled }}
```

Comment:
I would allow providing a configmap with the configuration

Reply:
+1 this option hasn't been implemented yet